### PR TITLE
Adding Logo image for itau (approved from bank)

### PR DESCRIPTION
**Pull request by shivapattada**

### CODE REVIEW

File: src/resources/fundingLogos/itau.svg [hunk 1]
```diff
@@ -1,34 +1,28 @@
...
+  <path fill="#FFFFFF" d="M 116.837 16.358 L 116.652 14.232 L 113.388 14.232 L 113.388 26.436 L 117.303 26.436 L 117.303 19.594 C 117.676 18.761 118.515 17.467 120.753 17.467 L 120.753 13.862 C 119.168 13.862 117.676 14.788 116.837 16.358 Z"/>
+  <path fill="#FFFFFF" d="M 131.384 8.87 L 131.384 15.25 C 130.73 14.602 129.704 13.862 127.653 13.862 C 124.017 13.862 121.592 17.097 121.592 20.703 C 121.499 23.939 124.204 26.621 127.467 26.713 C 128.958 26.621 130.451 26.157 131.663 25.326 L 132.222 26.528 L 135.3 26.528 L 135.3 8.962 L 131.384 8.962 Z M 128.679 23.569 C 126.349 23.569 125.881 21.627 125.881 20.148 C 125.881 18.669 126.535 16.912 128.679 16.912 C 129.985 16.912 130.824 17.838 131.384 18.669 L 131.384 21.627 C 130.824 22.646 129.892 23.569 128.679 23.569 Z"/>
+  <polygon fill="#FFF100" points="158.893 12.915 157.831 12.915 158.715 11.991 159.954 11.991"/>
+  <line fill="#FFFFFF" x1="35.897" y1="6.794" x2="35.897" y2="26.687"/>
+  <rect x="21.67" y="30.704" width="0.111" height="0.106" style="fill: rgb(216, 216, 216);"/>
```
Comment (bluepnume) [inline src/resources/fundingLogos/itau.svg]: Still need to remove style tags please

File: src/resources/fundingLogos/itau.svg [hunk 2]
```diff
@@ -1,34 +1,28 @@
...
+  <g transform="matrix(0.379173, 0, 0, 0.397443, 9.686229, 4.778377)" opacity="0.2">
+    <path fill="#231f20" d="M39.3 16.7c0.9 0.5 1.7 1.1 2.3 1.8 1 1.1 1.6 2.5 1.9 4.1 0.3-3.2-0.2-5.8-1.9-7.8-0.6-0.7-1.3-1.2-2.1-1.7C39.5 14.2 39.5 15.4 39.3 16.7z"/>
+    <path fill="#231f20" d="M0.4 45.2L6.7 5.6C6.8 4.5 7.8 3.7 8.9 3.7h16c5.5 0 9.8 1.2 12.2 3.9 1.2 1.4 1.9 3 2.2 4.8 0.4-3.6-0.2-6.1-2.2-8.4C34.7 1.2 30.4 0 24.9 0H8.9c-1.1 0-2.1 0.8-2.3 1.9L0 44.1C0 44.5 0.1 44.9 0.4 45.2z"/>
+    <path fill="#231f20" d="M10.7 49.4l-0.1 0.6c-0.1 0.4 0.1 0.8 0.4 1.1l0.3-1.7H10.7z"/>
+  </g>
+  <rect x="35.664" y="5.207" width="1.115" height="21.481" style="fill: rgb(255, 255, 255);" transform="matrix(0.999993, 0, 0, 1, 0.163238, -0.00471)"/>
```
Comment (bluepnume) [inline src/resources/fundingLogos/itau.svg]: Same

**Comment by shivapattada** [comment]
retest this please

**Comment by bluepnume** [comment]
Can you please check in any new screenshots in `test/screenshots/images`. That's what's causing the tests to fail.

**Comment by bluepnume** [comment]
Please regenerate screenshots and check them in. Looks like you have a conflict.

**Pull request closed without merging**